baseapp/grpcrouter.go

```diff
@@ -16,9 +21,11 @@ import (
 
 // GRPCQueryRouter routes ABCI Query requests to GRPC handlers
 type GRPCQueryRouter struct {
-	routes      map[string]GRPCQueryHandler
-	cdc         encoding.Codec
-	serviceData []serviceData
+	routes               map[string]GRPCQueryHandler
+	handlerByMessageName map[string][]func(ctx context.Context, req, resp protoiface.MessageV1) error
+	sdkCodec             codec.BinaryCodec
+	cdc                  encoding.Codec
```

Comment:
nit: Can we comment the fields? Explain what each codec is used for.

Also, consider renaming to `binaryCdc`

Reply:
Thank you, done!